src/components/Chart/Table.js

```diff
@@ -0,0 +1,233 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+
+import { css } from 'glamor'
+import { scaleThreshold, scaleOrdinal } from 'd3-scale'
+import { descending, ascending } from 'd3-array'
+
+import { useColorContext } from '../Colors/ColorContext'
+import { getFormat, getTextColor, deduplicate } from './utils'
+import { ExpandMoreIcon, ExpandLessIcon } from '../Icons'
+import { defaultProps } from './ChartContext'
+import { sansSerifRegular18 } from '../Typography/styles'
+import { PADDING } from '../Center'
+import { getColorMapper } from './colorMaps'
+
+const styles = {
+  container: css({
+    overflowX: 'auto',
+    overflowY: 'hidden',
+    marginLeft: -PADDING,
+    marginRight: -PADDING
+  }),
+  table: css({
+    ...sansSerifRegular18,
+    lineHeight: '1.2',
+    minWidth: '100%',
+    borderSpacing: '0 2px',
+    borderCollapse: 'separate',
+    padding: `0 ${PADDING - 10}px` // 10 cell padding on first and last cell in a row
+  }),
+  header: css({
+    borderBottomWidth: '1px',
+    borderBottomStyle: 'solid',
+    padding: '8px 10px',
+    userSelect: 'none'
+  }),
+  cell: css({
+    padding: '8px 10px'
+  }),
+  cellNumber: css({
+    textAlign: 'right',
+    fontFeatureSettings: '"tnum", "kern"'
+  })
+}
+
+const Table = props => {
+  const [colorScheme] = useColorContext()
+  const {
+    values,
+    numberFormat,
+    colorRanges,
+    colorRange,
+    defaultSortColumn,
+    thresholds,
+    tableColumns
+  } = props
+  const columns = values.columns
```

Comment:
One more thing:
```suggestion
  const columns = values.columns || Object.keys(values[0] || {})
```

This is for the case that values was not parsed with `d3-dsv` which provides `.columns` or is empty array like the default prop.